packages/data/README.md

```diff
@@ -374,6 +386,7 @@ function createCustomStore() {
 registry.registerGenericStore( 'custom-data', createCustomStore() );
 ```
 
+Note in the above example the handling of the optional `context` argument to `getSelectors()`. When using `withSelect`, this provides information about which component is requesting which data from the store. The benefit of this is the ability to keep track of which data is still currently needed by currently mounted components and which data is no longer needed.
```

Comment:
>The benefit of this is the ability to keep track of [...] which data is no longer needed

How would this be achieved? In the above example, the cache of `componentResources` will grow infinitely (which in its own right is very problematic). A component will not be removed if it's unmounted, nor is it obvious how this would be accomplished.

Reply:
The implementation I have for wc-admin removes components from the map upon unmount. The only components in the map are ones currently mounted at any given point in time.